<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MidiCmd</class>
 <widget class="QWidget" name="MidiCmd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MusE: Modify Gate Time</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rangeGroup" >
     <property name="title" >
      <string>Apply To</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="allEventsButton" >
        <property name="text" >
         <string>All Events</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectedEventsButton" >
        <property name="text" >
         <string>Selected Events</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="loopedEventsButton" >
        <property name="text" >
         <string>Looped Events</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectedLoopedButton" >
        <property name="text" >
         <string>Selected AND Looped</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
